README.rst

```diff
@@ -979,9 +993,20 @@ Choose from the list of available rules:
 
   Docblocks should have the same indentation as the documented subject.
 
-* **phpdoc_inline_tag** [@Symfony]
+* **phpdoc_inline_tag**
+
+  Fix phpdoc inline tags, make inheritdoc always inline (deprecated, use
+  ``phpdoc_inline_tag_normalizer`` and ``general_phpdoc_tag_rename``
+  instead).
 
-  Fix phpdoc inline tags, make inheritdoc always inline.
+* **phpdoc_inline_tag_normalizer** [@Symfony]
+
+  Fixes PHPDoc inline tags.
+
+  Configuration options:
+
+  - ``inheritdoc_always_inline`` (``bool``): whether to force ``@inheritDoc`` tag to
```

Comment:
I'm going to rename this option to `force_inheritdoc_type` and make it accept three values:
- `inline`: always make the `@inheritDoc` tag inline;
- `annotation`: always make the `@inheritDoc` tag an annotation;
- `none` (default): don't force a type for `@inheritDoc` tags (but still fix the inline ones).

WDYT?

Reply:
```php
[
    'tags' => [
        'inheritdoc' => 'inline' | 'annotation',
        'any_other_tag' => 'inline' | 'annotation',
    ],
]
```

maybe it shall be yet another fixer?
so fixer to manage given tags to be converted to inline or annotation,
and fixer to normalize inline tags ?

Reply:
wdyt @julienfalque ?

Reply:
Actually I think I implemented your suggestion and just forgot to answer here.